airbyte-protocol/models/src/main/resources/protocol_models/airbyte_message.yaml

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
can we make the resource path this: 
`airbyte-protocol/models/src/main/resources/airbyte_protocol_models/airbyte_message.yaml`?

if two modules have the same relative paths in resources, then one will clobber the other, so they need to be unique.